Access API/AccessAPI/DAL/VADAL.cs

```diff
@@ -1,9 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using System.Net;
-using Newtonsoft.Json;
+﻿using System.Net;
 using System.Text;
 
 
```

Comment:
Ikke her men på linje 11 siger check warning ""Converting null literal or possible null value to non-nullable type.""